client/components/post-excerpt/style.scss

```
@@ -33,7 +33,7 @@
 			right: 0;
 			width: 50%;
 			height: 24px;
-			background: linear-gradient(to right, rgba(255, 255, 255, 0), rgba(255, 255, 255, 1) 50%);
+			background: linear-gradient(to right, rgba(255, 255, 255, 0), var(--color-neutral-0) 50%);
```

Minor nit, although it may be hard to see visually. But we may want to start the gradient from the background instead of pure white:

```suggestion
			background: linear-gradient(to right, rgba(var(--color-neutral-0-rgb), 0), var(--color-neutral-0) 50%);
```